framework/core/image.cpp

```diff
@@ -194,5 +197,18 @@ std::unordered_set<ImageView *> &Image::get_views()
 	return views;
 }
 
+VkDeviceSize Image::get_image_size() const
```

Comment:
minor quibble but I would call this `get_image_allocated_size` or something like that.  Anything to avoid someone mistaking this for being a useful value to, for instance, determine how big a staging buffer needs to be to copy from an uncompressed source to the image.

Reply:
Addressed in my last commit